packages/quick_actions/quick_actions/ios/Classes/FLTQuickActionsPlugin.m

```diff
@@ -10,7 +10,9 @@ @interface FLTQuickActionsPlugin ()
 @property(nonatomic, retain) FlutterMethodChannel *channel;
 @end
 
-@implementation FLTQuickActionsPlugin
+@implementation FLTQuickActionsPlugin {
+  NSString *shortcutType;
```

Comment:
nits: Let's make this a property instead of ivar to match the code style of this file.

Reply:
done